### ISSUE TITLE
Allow placing the panel on the left

### ISSUE THREAD

**Issue by costincaraivan** [Microsoft/vscode#46402]
The panel manages to snatch defeat from the jaws of victory 😋

It has these configuration options: 

```
"workbench.panel.defaultLocation":  "bottom" / "right".
```

However, it can't be placed on the left side. I generally work in a setup where I have 2 windows side by side: the terminal on the left and the editor on the right.
Unfortunately this layout cannot be reproduced with the current Visual Studio Code settings 😞

**Comment by usernamehw** [comment]
Duplicate of #37877 or blocked by it.
